platform/mbed_version.h

```diff
@@ -0,0 +1,71 @@
+
```

Comment:
How is this file generated for each minor/patch release?

Reply:
@adbridge does that, she will have to update the release scripts